docker/Dockerfile

```diff
@@ -45,14 +45,20 @@ COPY docker/odbcinst.ini /etc
 # so `pip install pyrfc` breaks if all deps are not already present.
 RUN pip install cython==0.29.24 --user
 
+ARG INSTALL_DATABRICKS=false
+
 # BEGIN Databricks source setup
-RUN apt install -y wget apt-transport-https && \
+RUN if [ "$databricks" = "true" ]; then \
```

Comment:
shouldn't this be changed to if INSTALL_DATABRICKS?

Reply:
[🐛 Fixed bug with inaccessible packages](https://github.com/dyvenia/viadot/pull/880/commits/f4469106af842a4fd1890e0430639a1aa827dcfc)

Reply:
1. You inserted this block in the middle of viadot set up, can you move it above or below it?
https://github.com/dyvenia/viadot/blob/f4469106af842a4fd1890e0430639a1aa827dcfc/docker/Dockerfile#L59-L73

3. Just use if/else here

https://github.com/dyvenia/viadot/blob/f4469106af842a4fd1890e0430639a1aa827dcfc/docker/Dockerfile#L76-L79

Reply:
[🎨 Improved Dockerfile format](https://github.com/dyvenia/viadot/pull/880/commits/85145a7d614d915f20bd60ec188cc241537fc9db)